openmdao/visualization/scaling_viewer/scaling_table.html

```diff
@@ -466,31 +472,31 @@ <h3>Jacobian Info</h3>
         .domain(wrtEntries)
         .padding(0.01);
 
-    if (is_top) {
-        svg.append("g")
-            .attr("transform", "translate(0," + height + ")")
-            .call(d3.axisBottom(x))
-            .selectAll("text")
-            .style("text-anchor", "end")
-            .attr("transform", "rotate(-45)")
-            .attr("dx", "-.5em")
-            .attr("dy", ".3em");
-    }
+    // if (is_top) {
```

Comment:
Is there a reason to leave these commented-out instead of just removing them?

Reply:
Nope.  Removed.